packages/perseus/src/widgets/group/group.testdata.ts

```diff
@@ -159,32 +159,24 @@ export const simpleGroupQuestion: PerseusRenderer = {
         "group 1": {
             graded: true,
             options: {
-                content: "[[☃ numeric-input 1]]",
+                content: "[[☃ expression 1]]",
```

Comment:
It's weird to me that test data for `renderer.test.ts` is in `group.test.ts`. Not your fault, maybe even likely that I did that. It's just weird.

Reply:
The interactive-graph project ended up creating a builder pattern for ig options. That'd be amazing to have for all Perseus widget options.  But instead I think we sometimes cheated and used renderer configs from the Renderer tests.